include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -312,6 +325,8 @@ struct OrtApi {
 	*/
   OrtStatus*(ORT_API_CALL* AddCustomOpDomain)(_Inout_ OrtSessionOptions* options, _In_ OrtCustomOpDomain* custom_op_domain)NO_EXCEPTION;
 
+  OrtStatus*(ORT_API_CALL* RegisterCustomOpsLibrary)(_Inout_ OrtSessionOptions* options, _In_ const char* library_path)NO_EXCEPTION;
```

Comment:
The correct invocation of this function relies on a contract with the library being registered. Can we document this contract here or elsewhere?

Reply:
Good idea, I put some comments in the header to describe it.